rules/helm/package.tmpl.rb

```diff
@@ -0,0 +1,70 @@
+require "fileutils"
+require "open3"
+
+# Variables expanded by Bazel.
+package_dir = "[[package_dir]]"
+multipath_sep = "[[multipath_sep]]"
+tarsStr = "[[tars]]"
+generatedStr = "[[generated]]"
+helm = "[[helm]]"
+output_tgz = "[[output_tgz]]"
+
+# Variables expanded by gomplate.
+git_commit_short = '{{ (ds "workspace_status").STABLE_GIT_COMMIT_SHORT }}'
+git_branch = '{{ (ds "workspace_status").STABLE_GIT_BRANCH }}'
+
+# Create the temporary build directory for joining all the pieces required for packaging.
+tmp_build_dir = File.join("tmp", "build")
+build_dir = File.join(tmp_build_dir, package_dir)
+FileUtils.mkdir_p(build_dir)
+
+# Copy the source static files to the temporary build directory.
+Dir.glob(File.join(package_dir, "**", "*")) do |file|
+  dest = File.join(tmp_build_dir, file)
+  ftype = File.ftype(file)
+  if ftype == "directory"
+    FileUtils.mkdir_p(dest)
+  else
+    file = File.readlink(file) if ftype == "link"
```

Comment:
```suggestion
    file = File.readlink(file) while File.ftype(file) == "link"
```

Actually shouldn't we completely resolve the link before checking if it's a directory?